openmdao/docs/api_translation.rst

```diff
@@ -0,0 +1,363 @@
+.. _`api_translation`:
```

Comment:
The only comment I have here is that in the current re-structure all the docs directly beneath our top-level index.rst have their own directories (e.g. docs/user_guide),  with index.rst, docs, and other related items inside.  This doc kind of violates that structure, but then again, it doesn't NEED a directory of sub-docs, I suppose?  In theory, each section could be a sub-doc in an index, to keep things in @JustinSGray 's "bite-size" docs directive?  Anyway, just something to think about.

Reply:
(Note, the above re-structure is not something you need to do...just thinking aloud, and if it's decided to do it, I could take care of it after it goes in.)

Reply:
Too late. already put it in its own directory.